README.md

```diff
@@ -58,7 +69,7 @@ Follow these steps to run your existing Playwright test suite with the service.
 
 ### Set up environment
 
-Ensure that the `PLAYWRIGHT_SERVICE_ACCESS_TOKEN` and `PLAYWRIGHT_SERVICE_URL` that you obtained in previous steps are available in your environment.
+Ensure that the `PLAYWRIGHT_SERVICE_URL` that you obtained in previous step is available in your environment.
```

Comment:
Should we add note for old users, they may have very old URL which is without workspaceId, that wont work ?

Reply:
Added